packages/desktop-client/src/components/util/AmountInput.tsx

```diff
@@ -74,8 +75,15 @@ export function AmountInput({
   }
 
   function onInputTextChange(val) {
-    setValue(val ? val : '');
-    onChangeValue?.(val);
+    const comma = val.match(commaFormat);
```

Comment:
We already have utilities for formatting e.g. `format` from `useFormat` hook. Can we try using that to format on input update/blur?

Reply:
I hope I understood you correctly.